apps/web/next.config.js

```diff
@@ -1,24 +1,13 @@
+/** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
-  experimental: {
```

Comment:
We don't need this or was it accidentally removed?

Reply:
We don't need this since Next 13 when `transpilePackages` was moved out of experimental